crypto/ethsecp256k1/ethsecp256k1.go

```diff
@@ -201,12 +201,20 @@ func (pubKey *PubKey) UnmarshalAminoJSON(bz []byte) error {
 	return pubKey.UnmarshalAmino(bz)
 }
 
```

Comment:
Unrelated to your changes, but should the comment on line 35 be 

```
// ----------------------------------------------------------------------------
// eth_secp256k1 Private Key
```

same applies to line 136